lib/workers/repository/config-migration/common.ts

```diff
@@ -0,0 +1,14 @@
+import type { RenovateConfig } from '../../../config/types';
+import * as template from '../../../util/template';
+
+const migrationBranchTemplate = '{{{branchPrefix}}}migrate-config';
+
+let configMigrationBranch: string | null = null;
```

Comment:
we can't save this, it can change for each repo and this won't reset between them.